tests/vmi_lifecycle_test.go

```diff
@@ -829,6 +829,8 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 			//store old kubevirt-config
 			BeforeEach(func() {
+				// arm64 does not support cpu model
+				tests.SkipIfARM64()
```

Comment:
Maybe we can rework this function a little and use the content of the comment `arm64 does not support cpu model` as the `Skip` explanation.